### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by V-Blaze**
In this branch, These are the significant changes made:

- Added a form for creating a new post and implemented the 'Create' method in the `Post` controller.
- Add functionality for adding a comment and a like to a post.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -6,4 +6,22 @@ def index
...
+      author: current_user,
+      title: params[:post][:title],
+      text: params[:post][:text]
+    )
+
+    if @post.save
+      redirect_to users_path
+    else
+      render :new
+    end
```
Comment (TadesseDev) [inline app/controllers/posts_controller.rb]: - [Optional] Good job creating the post on behalf of the currently logged-in user. But I would suggest adding a `params method` to make your app cleaner and more secure. [Read more here](https://www.akshaykhot.com/why-use-strong-parameters-in-rails/)

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      text: params[:text]
+    )
+
+    if @comment.save
+      redirect_to user_posts_path(current_user)
+    else
+      render :new
+    end
+  end
+end
```
Comment (TadesseDev) [inline app/controllers/comments_controller.rb]: - [Optional] Good job creating the comment on behalf of the currently logged-in user. But I would suggest adding a `params method` to make your app cleaner and more secure. [Read more here](https://www.akshaykhot.com/why-use-strong-parameters-in-rails/)

**Review by TadesseDev — APPROVED** [review]
### Hi @V-Blaze 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ]  Correct GitFlow. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (V-Blaze) [reply app/controllers/comments_controller.rb]: Thanks for the suggestions, I have gone through the article and implemented the methods

**Pull request merged**